doc/funcref.sgml

```diff
@@ -8516,14 +8518,20 @@ if the mode is not supported due to lack of hardware.
 <descrip>
 <tag/Function/Wait until the start of the next video frame.
 <tag/Header/<tt/
-<ref id="apple2enh.h" name="apple2enh.h">,
+<ref id="apple2.h" name="apple2.h">,
 <ref id="atmos.h" name="atmos.h">,
 <ref id="cbm.h" name="cbm.h">,
 <ref id="gamate.h" name="gamate.h">,
 <ref id="nes.h" name="nes.h">,
 <ref id="pce.h" name="pce.h">/
-<tag/Declaration/<tt/void waitvsync (void);/
+<tag/Declaration/
+<tt>void waitvsync (void);</tt><newline>
+<tt>signed char waitvsync (void); /* For Apple II */</tt><newline>
 <tag/Description/Wait for vertical sync, to reduce flickering.
+<tag/Notes/<itemize>
+<item>The function will return -1 when the feature is not supported,
+like on the Apple&nbsp;&rsqb;&lsqb;+.
+</itemize>
```

Comment:
I think this requires some more work. So far waitvsync is typed as `void` everywhere. I believe that this needs to be changed globally - because it is desirable to be able to write cross-target code that checks for the waitvsync() return value. But it doesn't stop here: All targets having a waitvsync need to be enhanced to return 0 in AX.
The alternative is of course to say: How is a program supposed to mitigate waitvsync() failing? Maybe it's after all more appropriate to have it fail silently(?)

Reply:
Good question! I'm fine with both. Of course having it fail silently and having a note in the doc for the specific ][+ situation is enough.